client/src/constants.js

```diff
@@ -1,4 +1,10 @@
 // @flow
+import districtJson from './slovensko/okresy-centers.json'
+import regionJson from './slovensko/kraje-centers.json'
```

Comment:
Please also rename these two files to english. Also I don't know where you've taken the data from, but wouldn't it be better not to use geographical centers, but the seats of regions and districts? I mean Trencin for Trencin region etc. What do you think?

Reply:
I got the shapes of the regions and districts from freemap or freewiki, i dont remember, then i calculated the centroids, because i did not want to do it manually, especially for the districts. Then manually changed trencin region because it is not even approximately convex. We can allways change the input data to the cities if we find such database somewhere (i didnt look so far, but i can)

Reply:
Ok, that;s really low prio, do not bother with it now.

Reply:
done